webroot/js/components/chat/chat-input.js

```diff
@@ -226,6 +244,12 @@ export default class ChatInput extends Component {
     if (CHAT_KEY_MODIFIERS.includes(key)) {
       this.modifierKeyPressed = false;
     }
+    if (key === ':' || key === ';') {
+      if (this.injectEmoji()) {
+        // value could have been changed, update char count
+        textValue = formField.textContent;
+      }
+    }
     this.setState({
       inputCharsLeft: CHAT_MAX_MESSAGE_LENGTH - textValue.length,
```

Comment:
I think you'll need to add `inputText: textValue` to update state as well here

Reply:
Done!

Reply:
On second thought, we shouldn't even change the textValue here. The emojis should not add/deduct anything from the text content itself.